src/config/mod.rs

```diff
@@ -48,17 +53,21 @@ impl Args {
             fix,
             report_fixed_violations,
             force_rebuild,
+            paths,
         } = self;
         Config {
             rule,
             all_standalone_rules,
             all_plugins,
             fix,
             report_fixed_violations,
+            paths,
             config_file_path: Some(config_file_path),
             rule_configurations: config_file_content.rules().collect(),
             rules_by_plugin_prefixed_name: Default::default(),
             force_rebuild,
+            single_fixing_pass: Default::default(),
```

Comment:
Maybe this got wired up to the rule tester to mimic the ESLint behavior of only running a single linting pass (in rule tests)?